profiles/bicycle.lua

```diff
@@ -544,5 +544,10 @@ function turn_function(turn)
       if not turn.source_restricted and turn.target_restricted then
           turn.weight = turn.weight + 3000
       end
+    if normalized_angle >= 0.0 then
```

Comment:
What about just setting `turn.weight = turn.duration` on line 543 and removing this code? I think this was a c&p bug.